frontend/packages/ceph-storage-plugin/src/queries/ceph-queries.ts

```diff
@@ -233,6 +237,15 @@ export const getPoolQuery = (poolNames: string[], queryName: string) => {
     [StorageDashboardQuery.POOL_UTILIZATION_THROUGHPUT_QUERY]: `(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
       '|',
     )}'}`,
+    [StorageDashboardQuery.POOL_COMPRESSION_SAVINGS]: `(ceph_pool_compress_under_bytes - ceph_pool_compress_bytes_used) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
+      '|',
+    )}'}`,
+    [StorageDashboardQuery.POOL_COMPRESSION_ELIGIBILITY]: `((sum(ceph_pool_compress_under_bytes > 0) / sum(ceph_pool_stored_raw and ceph_pool_compress_under_bytes > 0)) * 100) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
```

Comment:
Do we really need SUM here, it is a single pool query so I doubting some is not makesense. May be i am wrong here, you better check with @anmolsachan once

Reply:
Corrected